### ISSUE TITLE
how to compare ckiplab/bert-base-chinese with bert-base-chinese?

### ISSUE THREAD

**Issue by bmkor** [ckiplab/ckip-transformers#17]
Thanks so much for this excellent model and having it accessible in huggingface.

Would like to know why the `ckiplab/bert-base-chinese` seems a bit strange to me when compared to the usual `bert-base-chinese` which I think it mainly trained on simplified chinese. For instance, when I masked the word `風` of the phrase `颱風預測。` in the usual `bert-base-chinese` it managed to give me back `風` with [high probability 0.992](https://huggingface.co/bert-base-chinese?text=%E9%A2%B1%5BMASK%5D%E9%A0%90%E6%B8%AC%E3%80%82); in contrast, in the `ckiplab/bert-base-chinese` it didn't give back the masked word `風` in the top 5 but [giving the word `的`](https://huggingface.co/ckiplab/bert-base-chinese?text=%E9%A2%B1%5BMASK%5D%E9%A0%90%E6%B8%AC%E3%80%82) with highest probability albeit only around 0.3 something which I am wondering. 

Is it supposed that we have to fine-tune this MLM first? Or perhaps I interpreted it wrongly (as I'm very new in this field). Mind sharing a bit on your thought? Thanks very much and thanks in advance.

**Comment by emfomy** [comment]
Based on the property of the training data and the task objective, this model performs poorly on short or incomplete sentences.
For longer sentences like [`新聞報導颱[MASK]預測。`](https://huggingface.co/ckiplab/bert-base-chinese?text=新聞報導颱%5BMASK%5D預測%E3%80%82), this model managed to give me back `風` in high probability.

Thanks for pointing out the weakness of our model. We will keep working to make it better.
